libexec/cli/help.exec

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# 
+# Copyright (c) 2017, SingularityWare, LLC. All rights reserved.
+# Copyright (c) 2015-2017, Gregory M. Kurtzer. All rights reserved.
+# Copyright (c) 2017, Vanessa Sochat. All rights reserved.
+
+## Basic sanity
+if [ -z "$SINGULARITY_libexecdir" ]; then
+    echo "Could not identify the Singularity libexecdir."
+    exit 1
+fi
+
+## Load functions
+if [ -f "$SINGULARITY_libexecdir/singularity/functions" ]; then
+    . "$SINGULARITY_libexecdir/singularity/functions"
+else
+    echo "Error loading functions: $SINGULARITY_libexecdir/singularity/functions"
+    exit 1
+fi
+
+RETVAL=0
+SINGULARITY_IMAGE="${1:-}"
```

Comment:
If no singularity image is defined at $1, we should either exit and error, print usage summary, or print the `.../libexec/cli/singularity.help`.